src/App.js

```diff
@@ -1,17 +1,197 @@
-import React, { Component } from 'react';
-import Letters from './components/Letters';
-import './App.css';
-import Score from './components/Score';
-
-class App extends Component {
-  render() {
-    return (
-      <div>
-        <Score />
-        <Letters />
-      </div>
-    );
+import React, { Component } from "react";
+import Letters from "./components/Letters";
+import Score from "./components/Score";
+import Solution from "./components/Solution";
+import "./App.css";
+// import Background from '../public/insideout.jpg';
+let words =  [
+  {word:"SATISFY",hint:"Awesome mood",secondHint:"Very good mood"},
+  {word:"HAPPINESS",hint:"Yellow mood",secondHint:"Look at the background"},
+  {word:"FEAR",hint:"Purple mood",secondHint:"Look at the background"},
+  {word:"ANGER",hint:"Red mood",secondHint:"Look at the background"},
+  {word:"DISGUSTE",hint:"Green mood",secondHint:"Look at the background"},
+  {word:"SADNESS",hint:"Blue mood",secondHint:"Look at the background"}];
+  let rand = Math.floor((Math.random() * words.length));
+  let toShow = false; //hide
+  
+  
+  
+  // create an array of letters
+  class App extends Component {
+    constructor() {
+      super();
+      this.state = {
+        letterStatus: this.generateLetterStatus(),
+        score: { value: 100, className: "high-score" },
+        word:  words[rand].word,
+        hint: words[rand].hint,
+        secondHint : words[rand].secondHint
+      };
+    }
+    
+    secondHint = ()=>{
+      toShow = false;
+      alert(this.state.secondHint);
+      this.setState({...this.state});
+    }
```

Comment:
this method doesn't do anything useful..?
again, if `toShow` affects render, it should be in `state`